pkg/grpc/universalapi/api_actors.go

```diff
@@ -47,3 +53,51 @@ func (a *UniversalAPI) WaitForActorsReady(ctx context.Context) {
 	case <-a.actorsReadyCh:
 	}
 }
+
+// This function makes sure that the actor subsystem is ready.
+func (a *UniversalAPI) actorReadinessCheck(ctx context.Context) error {
```

Comment:
Looks like this was copy-pasted from pkg/grpc/api.go... Please don't copy/paste code as that makes it harder to maintain it.

Instead, make this method exported, and then change pkg/grpc to use "actorReadinessCheck" from the Universal property